### DISCUSSION TITLE
Can't set "Files" as alternative file manager

### DISCUSSION THREAD

**Discussion by keppn** [adirh3/Fluent-Search#748] [Q&A]
I just found out about Fluent Search - and I'm flabbergasted so far! Very nice app <3

A little thing I can't get working is an alternate file manager. I'd like to use the popular "Files" app. 
I specified the directory, but Fluent Search still opens folders with Windows Explorer. 
![image](https://user-images.githubusercontent.com/121324132/209416738-1bda7c26-b5a0-4da8-9db1-92ab816dc02d.png)

Do I need to fill in something in "arguments"? I'm not sure what that is about :D 
Or is this about "Files" being an UWP app? 

Thanks for helping out!

**Comment by adirh3** [comment]
To set Files as default file manager set the `File path` to `files` and arguments to `"%1"`.
Also, clicking on the suggestions icon will show you detected file managers by Fluent Search and you can click on any to set them -
![image](https://user-images.githubusercontent.com/27368554/209433994-d820506e-6151-4d67-a92f-6511c9a86c1c.png)

**Reply by keppn** [reply]
Thank you very much, %1 as argument works. 
The suggestions aren't populated in my case, that I would have figured out :) 
Great support, thanks again <3
